introduction/08-create-custom-boilerplate.rst

```diff
@@ -0,0 +1,512 @@
++-----------------------------+----------------------------------------+------------------------+
+| application base template   | ``polls/templates/polls/base.html``    | overridden by ↓        |
++-----------------------------+----------------------------------------+------------------------+
+| application base template   | ``templates/polls/base.html``          | overrides ↑, extends ↓ |
++-----------------------------+----------------------------------------+------------------------+
+| project-specific template   |``templates/main.html``                 | extends ↓              |
++-----------------------------+----------------------------------------+------------------------+
+| generic Foundation template | ``templates/base.html``                |                        |
++-----------------------------+----------------------------------------+------------------------+
+
+
+You don't have to remember all this, or even understand it fully right now -
+but it's here if you need to refer to it.
+
+**Why this structure?**
+
+Keeping the generic Foundation template free of any project-specific material
+will make it easier to use in other projects. Keeping application-specific
+material out of project templates will make it easier to use them with other
+applications.
+
+
+.. _create-boilerplate-package:
+
+Create the Boilerplate package
+------------------------------
+
+We now have enough for a basic, working Boilerplate. It provides:
+
+* a ``base.html`` Foundation template that is replete with ``{% block %}``
+  template tags, allowing it to be extended in a vast variety of ways
+* a ``main.html`` template that the project developer can customise
+* Foundation's static CSS and JS assets.
+
+For convenience, we will create a new directory called ``tutorial-boilerplate``
+in the root of the project, and **copy** those items to it, so that the
+directory looks like this::
+
+    tutorial-boilerplate/
+        static/
+            css/
+            js/
+        templates/
+            base.html
+            main.html
+
+
+The ``boilerplate.json`` file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a ``boilerplate.json`` in ``tutorial-boilerplate``:
+
+..  code-block:: JSON
+
+    {
+        "package-name": "tutorial-boilerplate",
+        "templates": [],
+        "identifier": "foundation6",
+        "version": "0.0.1"
+    }
+
+* The ``package-name`` is whatever you'd like to call it - however, it must
+  be unique on the Divio Cloud system.
+* ``templates`` are only required for Boilerplates intended to be used with
+  django CMS.
+* ``identifier`` is a namespace, that will allow applications that are
+  Boilerplate-aware to build in support for particular Boilerplates into their
+  own frontend code. An example of this is `Aldryn News & Blog
+  <https://github.com/aldryn/aldryn-newsblog/tree/master/aldryn_newsblog>`_.
+* The ``version`` is for your own convenience.
+
+Run the ``boilerplate verify`` command to check that the ``boilerplate.json``
```

Comment:
i'm personally a bit confused about `boilerplate verify`, it's not a real command but is marked like one

Reply:
Is it verify or validate? underneath is validate